hs_core/templates/pages/genericresource.html

```diff
@@ -2651,15 +2707,20 @@ <h3 style="color: red" id="file-type-error"></h3>
                                 rowTemplate.find("span[data-col='current-access']").text("Is owner");
                                 rowTemplate.find("span[data-col='current-access']").append(" <span class='caret'></span>");
                                 rowTemplate.find(".share-form-owner").parent().addClass("active");
-                                // Enable dropdown menu for other owner entries
-                                $(".access-table li.active[data-access-type='Is owner']").closest("tr").removeClass("hide-actions");
                             }
                             $(".access-table tbody").append($("<tr id='row-id-" + share_with + "'>" + rowTemplate.html() + "</tr>"));
+
+                            updateActionsState(json_response.current_user_privilege);
                         }
                         else if (json_response.status == "error") {
                             $("#div-invite-people").append("<span class='label label-danger'><strong>Error: </strong>" + json_response.error_msg + "</span>");
                         }
                         setPointerEvents(true);
+                    },
+                    error: function(XMLHttpRequest, textStatus, errorThrown){
+                        $("#div-invite-people").find(".label-danger").remove(); // Remove previous alerts
```

Comment:
Same comment as above to make sure find() does not return null when calling remove() on it